### DISCUSSION TITLE
gradle/2022/08/28/how-to-publish-gradle-plugin/

### DISCUSSION THREAD

**Discussion by giscus** [Moosphan/Moosphan.github.io#9] [Comments]
# gradle/2022/08/28/how-to-publish-gradle-plugin/

修技术之道，日三省吾身 | App Dev & Freelance Thinker & Hacker Growth | 这里是 @道轲(Dorck) 的个人博客，与你一起探寻世界。

https://dorck.cn/gradle/2022/08/28/how-to-publish-gradle-plugin/

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。
